codewindtechpreview.md

```diff
@@ -45,7 +45,7 @@ Projects must be located in `<user>/microclimate-workspace`.
 
 #### Downloading the release binary on MacOS
 1. Download <a href="download/codewind" class="download-link trackdownload">codewind</a> to a folder on your system.
-2. Make the `codewind` file usable with the `chmod 775 codewind` command.
+2. Make the `codewind` file usable with the `chmod +r codewind` command.
```

Comment:
`chmod -r` removes all permissions.  I think you want `chmod +x`

Reply:
I will update accordingly.